torchaudio/csrc/sox_effects.cpp

```diff
@@ -1,41 +1,48 @@
 #include <sox.h>
 #include <torchaudio/csrc/sox_effects.h>
 
-using namespace torch::indexing;
-
 namespace torchaudio {
 namespace sox_effects {
 
 namespace {
 
 enum SoxEffectsResourceState { NotInitialized, Initialized, ShutDown };
 SoxEffectsResourceState SOX_RESOURCE_STATE = NotInitialized;
+std::mutex SOX_RESOUCE_STATE_MUTEX;
```

Comment:
So the code is thread through this global lock, right?

Reply:
> So the code is thread safe through this global lock, right?

Yes.